docs/source/markdown/podman-volume-create.1.md

```diff
@@ -55,7 +55,9 @@ The `o` option sets options for the mount, and is equivalent to the filesystem
 options (also `-o`) passed to **mount(8)** with the following exceptions:
 
   - The `o` option supports `uid` and `gid` options to set the UID and GID of the created volume that are not normally supported by **mount(8)**.
-  - The `o` option supports the `size` option to set the maximum size of the created volume, the `inodes` option to set the maximum number of inodes for the volume and `noquota` to completely disable quota support even for tracking of disk usage. Currently these flags are only supported on "xfs" file system mounted with the `prjquota` flag described in the **xfs_quota(8)** man page.
+  - The `o` option supports the `size` option to set the maximum size of the created volume, the `inodes` option to set the maximum number of inodes for the volume and `noquota` to completely disable quota support even for tracking of disk usage.
+  The `size` and `inodes` options are supported on "xfs" file system mounted with the `prjquota` flag described in the **xfs_quota(8)** man page.
+  The `size` option is supported on "tmpfs" file systems.
```

Comment:
I'm having a hard time with this. I find this more understandable, but am not sure if it's correct:
```
The `size` option is supported only on `tmpfs` and `xfs[note]` filesystems.
The `inodes` option is supported only on `xfs[note]`. Any other use will throw an error, maybe, I dunno?
Note: xfs filesystems must be mounted with the `prjquota` flag as described in **xfs_quota(8)**. Podman will throw an error if they're not.
```